dsc/src/control.rs

```diff
@@ -373,6 +406,29 @@ async fn dsc_enable_restart_all(
     Ok(HttpResponseUpdatedNoContent())
 }
 
+/**
+ * Fetch the region info for our downstairs
+ */
+#[endpoint {
+    method = GET,
+    path = "/regioninfo",
+}]
+async fn dsc_get_region_info(
+    rqctx: RequestContext<Arc<DownstairsControl>>,
+) -> Result<HttpResponseOk<RegionExtentInfo>, HttpError> {
+    let api_context = rqctx.context();
+
+    let region_info =
+        api_context.dsci.get_region_info().await.map_err(|e| {
+            HttpError::for_bad_request(
+                None,
+                format!("failed to state for downstairs {}: {:#}", 0, e),
```

Comment:
```suggestion
                format!("failed to get region info for downstairs {}: {:#}", 0, e),
```

For bonus points, this could also be fixed in `dsc_get_pid` ("failed to get PID") and `dsc_get_ds_state` ("failed to get state" instead of "failed to state")

Reply:
And, there is no downstairs whom we are specifically asking either.  I'll fix them all

Reply:
Fixed in 8acd4d6

Reply:
almost fixed in 30e467f, no actual diff for 8acd4d6, but then really fixed in f5cf0d1